app/webpacker/styles/application.scss

```diff
@@ -38,3 +38,18 @@ a[href="#"] {
     font-weight: $govuk-font-weight-bold;
   }
 }
+
+
+// Dashed list (used in Contents links list)
+
+.govuk-list--dash li {
```

Comment:
Should this be `app-list--dash` ?

Reply:
👍 need to update the prototype its also uses this class name